spec/oca-epak/oep/client_spec.rb

```diff
@@ -33,4 +33,19 @@
       end
     end
   end
+
+  describe "#get_pdf_de_etiquetas_por_orden_or_numero_envio" do
+    let(:real_order) { 21132466 }
+    let(:fake_order) { 123_000_000 }
```

Comment:
@cavi21 `fake_order` isn't getting used. Could you add a scenario where there is a `fake_order`? Just to see that we can handle a response where the PDF is not found. 

It would look like this: 

```
result = subject.get_pdf_de_etiquetas_por_orden_or_numero_envio(id_orden_retiro: fake_order)
expect(result).not_to be # not sure if this is the best idea
```